content/Store Support/Shipping/About-the-delivery-process.md

```diff
@@ -87,18 +87,11 @@ If you want to return your order and receive a refund you can reject to receive
 
 ---
 
-<a id="if-a-parcel-did-not-arrive"></a>
+<a id="if-you-havent-received-your-delivery"></a>
 
-## If a parcel did not arrive
+## If you haven't received your delivery
 
-In some rare cases, an order might not be delivered to the customer even if it appears as delivered when tracking the parcel. This mostly happens when using shipping methods that don't have proof of delivery.
-
-If the parcel did not arrive, please check the following before contacting [Arduino's customer support](https://www.arduino.cc/en/contact-us/):
-
-* Check if other people living in the same delivery address or a neighbor have picked up the parcel.
-* Check around the entrance(s) of your residence, particularly on back porches, bushes, garages, grills, or other places that might protect your package from theft or weather.
-
-> Note that carriers may also use plastic bags to protect your package from the weather.
+If you haven't received your order yet, please click [here](https://support.arduino.cc/hc/en-us/articles/14085643116700-If-you-haven-t-received-your-delivery) for more information.
```

Comment:
```suggestion
[If you haven't received your delivery](https://support.arduino.cc/hc/en-us/articles/14085643116700-If-you-haven-t-received-your-delivery){.link-chevron-right}
```
Using the chevron style to stay consistent with the [Track your order section](https://github.com/arduino/help-center-content/blob/df9ef3557c2e02d1096139118a1a704953d1e7c7/content/Store%20Support/Shipping/About-the-delivery-process.md?plain=1#L45-L47).